assets/styles/components/_wysiwyg.scss

```diff
@@ -0,0 +1,363 @@
+    img,
+    table {
+        margin-bottom: rem(30px);
+        & + p,
+        & + ul,
+        & + ol {
+            margin-top: 1em;
+            margin-bottom: rem(30px);
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: rem(60px);
+        }
+    }
+
+    ul, ol, p {
+        line-height: 1.5;
+    }
+
+
+    // ==========================================================================
+    // Headings
+    // ==========================================================================
+    b, strong {
+
+    }
+
+    h1,h2,h3,h4,h5,h6 {
+        // @include heading;
+        font-weight: normal;
+    }
+
+    h1 {
+        // @include heading-h3;
+        margin-top: size-clamp('lg');
+        margin-bottom: rem(40px);
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: rem(40px);
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h2 {
+        // @include heading-h3;
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('md');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('md');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h3 {
+        // @include heading-h4;
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('md');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('md');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h4,
+    h5,
+    h6 {
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('sm');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('sm');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('sm');
+        }
+    }
+
+    h4 {
+        // @include heading-h5;
+    }
+
+    h5 {
+        // @include heading-h6;
+    }
+
+    h6 {
+        // @include heading-h6;
+    }
+
+
+    // ==========================================================================
+    // Lists
+    // ==========================================================================
+    ul,
+    ol {
+        position: relative;
+        margin: size-clamp('md') 0;
+
+        >li {
+            padding-left: rem(30px);
+        }
+    }
+
+    ul {
+        & > li {
+            position: relative;
```

Comment:
> `position: relative;`

On `blockquote`, `li`, `ol`, and `ul` seems like its just asking for trouble.

Reply:
We usually remove the default browser style for these elements and create our own using absolute positionned pseudo elements. In this context of default wysiwyg settings, we can use the default browser styling and overwrite if needed.
I removed the relative position on these 4 elements.